docs/quickstart.rst

```diff
@@ -61,198 +199,276 @@ instance had already existing in the destination directory, this command
+operating even if the Postgres primary goes down. We'll install it as a service
+with systemd so that it will resume if the VM restarts.
 
 .. code-block:: bash
 
-  pg_autoctl run --pgdata /mnt/node_a/data
+   ssh -l ha-admin `vm_ip a` << CMD
+     sudo -i -u postgres \
+       pg_autoctl -q show systemd --pgdata ~ha-admin/ha > pgautofailover.service
+     sudo mv pgautofailover.service /etc/systemd/system
+     sudo systemctl daemon-reload
+     sudo systemctl enable pgautofailover
+     sudo systemctl start pgautofailover
+   CMD
 
-This will remain running in the terminal, outputting logs. We can open
-another terminal and start a secondary database similarly to how we
-created the primary:
+Next connect to node B and do the same process. We'll do both steps at once:
 
 .. code-block:: bash
 
-   pg_autoctl create postgres  \
-     --pgdata ./node_b      \
-     --pgport 6011          \
-     --pgctl /usr/pgsql-11/bin/pg_ctl \
-     --monitor postgres://autoctl_node@127.0.0.1:6000/pg_auto_failover
+   ssh -l ha-admin `vm_ip b` -- \
+     pg_autoctl create postgres \
+       --pgdata ha \
+       --auth trust \
+       --ssl-self-signed \
+       --username ha-admin \
+       --dbname appdb \
+       --nodename ha-demo-b.internal.cloudapp.net \
+       --pgctl /usr/lib/postgresql/11/bin/pg_ctl \
+       --monitor postgres://autoctl_node@ha-demo-monitor.internal.cloudapp.net/pg_auto_failover?sslmode=require
+
+   ssh -l ha-admin `vm_ip b` << CMD
+     sudo -i -u postgres \
+       pg_autoctl -q show systemd --pgdata ~ha-admin/ha > pgautofailover.service
+     sudo mv pgautofailover.service /etc/systemd/system
+     sudo systemctl daemon-reload
+     sudo systemctl enable pgautofailover
+     sudo systemctl start pgautofailover
+   CMD
+
+It discovers from the monitor that a primary exists, and then switches its own
+state to be a hot standby and begins streaming WAL contents from the primary.
+
+Node communication
+------------------
 
-   pg_autoctl run --pgdata ./node_b
+For convenience, pg_autoctl modifies each node's ``pg_hba.conf`` file to allow
+the nodes to connect to one another. For instance, pg_autoctl added the
+following lines to node A:
 
-All that differs here is we’re running it on another port and pointing
-at another data directory. It discovers from the monitor that a primary
-exists, and then switches its own state to be a hot standby and begins
-streaming WAL contents from the primary.
+.. code-block:: ini
+
+   # automatically added to node A
+
+   host "appdb" "ha-admin" ha-demo-a.internal.cloudapp.net trust
+   host all "pgautofailover_monitor" ha-demo-monitor.internal.cloudapp.net trust
+   host replication "pgautofailover_replicator" ha-demo-b.internal.cloudapp.net trust
+   host "appdb" "pgautofailover_replicator" ha-demo-b.internal.cloudapp.net trust
+   host replication "pgautofailover_replicator" ha-demo-a.internal.cloudapp.net trust
+   host "appdb" "pgautofailover_replicator" ha-demo-a.internal.cloudapp.net trust
```

Comment:
At this point in the tutorial the last two lines have not been added yet, because we didn't failover. The HBA editing is “lazy” and only happens when needed. I think we should mention that too, and encourage our users to edit the HBA anyway they see fit, because it's going to be needed for the application anyways.